src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -66,6 +68,7 @@ public static AddressBook getTypicalAddressBook() {
         AddressBook ab = new AddressBook();
         for (Person person : getTypicalPersons()) {
             ab.addPerson(person);
+            System.out.println(person);
```

Comment:
Another debug statement to be removed.

Reply:
Yes, will remove it, thanks for discovering it 👍